proto/money/rpc/master_folio_svc.proto

```diff
@@ -40,10 +41,24 @@ message MasterFolioServiceRemoveMasterFolioResponse {
 	ReservationFolioSvcRemoveMasterFolioResult result = 1;
 }
 
+enum MasterFolioSvcFolioSwapResult {
+	MASTER_FOLIO_SVC_FOLIO_SWAP_OK = 0;
+	MASTER_FOLIO_SVC_FOLIO_SWAP_IVALID_NO_MASTER_FOLIO = 1;
+	MASTER_FOLIO_SVC_FOLIO_SWAP_INVALID = 2;
```

Comment:
I generally like having in these results a "catch-all" failure case. The above is for the specific case that there isn't an established client-master relationship for performing the swap. There are a bunch more cases though why a swap could not happen, and I didn't want to try to enumerate all of them, especially when they could be caught by the UI being a bit attentive about what requests it makes.